routes/index.js

```
@@ -58,7 +58,7 @@ module.exports = function (app, io, nconf, parallax, usernamesDb, crypto, Parall
             });
           });
         } catch (e) {
-          console.err(e);
+          //console.err(e);
```

Hm do we really just want to swallow errors?